geonode_mapstore_client/client/static/mapstore/configs/localConfig.json

```diff
@@ -281,6 +281,25 @@
                     }
                 ]
             },
+            "actions":{
```

Comment:
We should use a workflow similar to the menu lists in BrandNavbar, MenuIndex and Footer so something like this:
```js
{
"items": [
    {
        "type": "dropdown",
        "authenticated": true,
        "labelId": "gnhome.createNew",
        "permissions": ['add_resoucesbase'], // to verify based on the  new endpoints
        "items": [
            {
                "type": "link",
                "href": "/maps/new",
                "labelId": "gnhome.map"
            },
            {
                "type": "link",
                "href": "/apps/new?appType=GeoStory",
                "labelId": "gnhome.geostory"
            }
        ]
    }
]
}
```